ads/google/a4a/utils.js

```diff
@@ -50,6 +50,17 @@ export const ValidAdContainerTypes = {
   'AMP-STICKY-AD': 'sa',
 };
 
+/**
+ * See `VisibilityState` enum.
+ * @const {!Object<string, string>}
+ */
+const visibilityStateCodes = {
+  'visible': '1',
```

Comment:
I am slightly tempted to suggest the following, but probably not worth it:
```
const visibilityStateCodes = {};
visibilityStateCodes[VisibilityState.VISIBLE] = 1;
...etc.
```
Also, why does this not include paused, inactive, and does include unloaded which doesn't appear in https://github.com/ampproject/amphtml/blob/c44a48fbb1dbd0de0b5a9e20f8f69bb920705dde/src/visibility-state.js#L22 ?

Reply:
These are the values of current interest.  Reworked so that object references VisibilityState values directly as you suggested.

Reply:
Re: dep-check revert: d'oh. Thanks for trying.